common/libs/validators.ts

```diff
@@ -84,11 +84,24 @@ function validateEtherAddress(address: string): boolean {
   }
 }
 
+export function fixPkey(key) {
+  if (key.indexOf('0x') === 0) {
+    return key.slice(2);
+  }
+  return key;
+}
+
 export function isValidPrivKey(privkey: string | Buffer): boolean {
   if (typeof privkey === 'string') {
-    return privkey.length === 64;
+    const initialCheck =
+      privkey.length === 64 && /^[a-fA-F0-9]+$/g.test(privkey);
```

Comment:
What is this regex checking in simple terms?

Is there an ethereumjs-library method we can use instead?

Reply:
this regex simply checks if the string is a valid hex.